### PR TITLE
added /kitty command

### PR DESCRIPTION
**Pull request by tastefulblatant**
I made a `/kitty` command, which sends a kitty review gif when executed. Oliverrr#8393 helped me with gathering the gifs.

### CODE REVIEW

**Comment by petulkauwu** [comment]
aint no way bro

**Comment by oliverrr123** [comment]
let's go

**Comment by tastefulblatant** [comment]
finally

**Comment by EquinoxAlpha** [comment]
wait a moment
image

File: src/commands/kitty_review.rs [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+        "https://media.tenor.com/Pj8H-4ee_MMAAAAM/cat-flexible.gif",
+        "https://media.tenor.com/cJkWMVSwAy4AAAAM/flavortown-monkey-review.gif",
+        "https://media.tenor.com/v-GggtEUBjsAAAAM/kitty-review-cowboy-bebop.gif",
+        "https://media.tenor.com/qpNAAIF7pHEAAAAM/kapi-kittyreview.gif",
+        "https://media.tenor.com/1P4gdfAXiXoAAAAM/cat.gif",
+        "https://media.tenor.com/njLU1utI8aIAAAAM/flavortown-monkey-review.gif",
+        "https://media.tenor.com/FreMmPqrvfgAAAAM/cat.gif",
+        "https://media.tenor.com/xb4fVIuelFsAAAAM/camerupt-camerupt-review.gif",
+        "https://media.tenor.com/KyVImKcVC5oAAAAM/flavortown-monkey-review.gif"
+    ];
```
Comment (marekvospel) [inline src/commands/kitty_review.rs:200]: I'd rather put this in a separate file, as it's really long.

File: crates/i18n/Cargo.toml [hunk 2]
```diff
@@ -12,3 +12,4 @@ serde_json = "1.0.92"
 anyhow = "1.0.69"
 quote = "1.0.23"
 proc-macro2 = "1.0"
+rand = "0.8.4"
```
Comment (marekvospel) [inline crates/i18n/Cargo.toml:15]: i18n crate doesn't need random lib

**Review by marekvospel — COMMENTED** [review]
Thanks for the contribution! The code looks good except for some minor problems. I won't merge this however, because I find the approach of #16 smarter, as there is no need to have a huge list of gifs and there is no need to check them, as the content on tenor should be managed by google.

**Comment by marekvospel** [comment]
By the way you can add `Co-Authored-By: Name <email>` to indicate someone helped you create it. It's aleo recognised by github, so it will also appear on their profile as a contribution.

https://stackoverflow.com/questions/7442112/how-to-attribute-a-single-commit-to-multiple-developers

**Pull request closed without merging**